offload_data/plugin.cpp

```diff
@@ -43,8 +47,8 @@ class offload_data : public plugin {
 
     virtual ~offload_data() override {
         // Write offloaded data from memory to disk
-        if (enable_offload) {
-            boost::filesystem::path p(obj_dir);
+        if (_m_enable_offload) {
+            boost::filesystem::path p(obj_dir); // maybe replace with _m_obj_dir
```

Comment:
Yes, `OFFLOAD_PATH` is defined in `configs/global_module_defs.yaml` as also `metrics/offloaded_data/`, so it's replaceable.